nativeApp/src/lib/ui/forms/input.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components/native'
+import { Fonts } from '../typography';
+import { ColoursEnum } from '../colours';
+
+const InputContainer = styled.View`
+  flexDirection: row;
+  alignItems: center;
+  marginTop: 20;
```

Comment:
20 what? we should make units explicit everywhere.

Reply:
This is a `react-native` thing. RN uses `Stylesheet`, which takes a styles object, the properties are similar to css but not actual css. Styled components wraps the stylesheet method and makes it look more like css but with camel case. 

TBH I am unsure atm if style components is actually useful with RN, and just _makes it look like_ we're working with css, but actually hides that it's functionality is different.